package.json

```diff
@@ -247,7 +247,7 @@
     "chai": "3.5.0",
     "chai-enzyme": "0.5.2",
     "deep-freeze": "0.0.1",
-    "enzyme": "2.4.1",
+    "enzyme": "2.9.1",
```

Comment:
As far as we can go with React 15.x 👍 

Migrating to React 16 test wise might be a big challenge.

Reply:
Think so? I was hoping to upgrade to `enzyme` 3.x once https://github.com/producthunt/chai-enzyme/pull/198 is merged, an a new `chai-enzyme` version that includes it is released...

Reply:
I forgot about enzyme adapter for React 15. It might be solid. I haven’t tried it. We get some failing tests in Gutenberg when trying to use enzyme 3 with React 16. It looks like it has some hidden bugs.

You can also explore [jest-enzyme](https://www.npmjs.com/package/jest-enzyme) 😃